app/controllers/users_controller.rb

```diff
@@ -49,7 +49,7 @@ def query_prs
       {
         title: pr.title,
         url: pr.html_url,
-        date: (pr.closed_at || pr.created_at).to_date,
+        date: (pr.created_at || pr.closed_at).to_date,
```

Comment:
Is it worth removing the `pr.closed_at` entirely given the search is only for `created:$FRIDAYS`?